package.json

```diff
@@ -8,26 +8,29 @@
         "clean": "rm -rf dist lib",
         "format": "prettier --write .",
         "format-check": "prettier --check .",
-        "build": "tsc",
-        "package": "ncc build --minify",
+        "lint": "eslint \"**/*.ts\" --fix",
+        "package": "ncc build src/index.ts -o dist/setup --minify && ncc build src/post.ts -o dist/cache-save --minify",
```

Comment:
Just to make sure I understand, the contents of `post.ts` are what's getting built into `dist/cache-save/index.js`?

Reply:
Yes, `-o dist/cache-save ` specifies the output directory and the default name is always `index.js`. We need this step to convert `post.ts` from typescript to javascript and "minify" everything.

We didn't have to specify as many options before because we were leveraging defaults but now we need to make 2 javascript files: one for the main action and one for the post step. I updated some other stuff in our package.json to be more similar to what the Github supported actions  are doing in these cases.